### ISSUE TITLE
Documentation - browserize.sh script and browser application examples require npm 3

### ISSUE THREAD

**Issue by gkwicker** [aws/aws-iot-device-sdk-js#53]
The browserize.sh script and browser application examples require npm 3.  The documentation will be updated to reflect this in a future release.  When using older versions of npm, the browserize.sh script will generate the following error and the bundle will not be usable:

``` sh
./scripts/browserize.sh: line 55: cd: browser/node_modules/mqtt/lib/connect: No such file or directory
```

**Comment by electronut** [comment]
Using npm3 doesn't seem to resolve this issue. See issue #74

**Comment by gkwicker** [comment]
Hi @electronut, have a look at [this post](https://github.com/aws/aws-iot-device-sdk-js/issues/74#issuecomment-241797679) to see if it solves your issue.  Thanks for using AWS IoT.

**Comment by liuszeng** [comment]
Addressed in release [v1.0.13](https://github.com/aws/aws-iot-device-sdk-js/releases/tag/v1.0.13).
